language/move-compiler/src/parser/parsed_tree/translate/translate_address.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
I'm not a fan of these separate modules/files for each section of the ast. I know the files get very large, but having everything in one place makes understanding the entire flow easier in my opinion. (plus it is then more consistent with the rest of the compiler)

Reply:
OK. Copy that.

Reply:
I was thinking I would migrate these files into one when the work is done(tests passed) because I am more comfortable with seperate files while developing. I planned to ask after the work is done. But nvm, I would definitely keep this consistent with other packages.